src/praisonai-agents/praisonaiagents/agents/agents.py

```
@@ -1609,25 +1609,25 @@ def verbose_task_complete_callback(task, task_output):
             original_on_task_start = self.on_task_start
             original_on_task_complete = self.on_task_complete
             
-            def composed_on_task_start(task):
+            def composed_on_task_start(task, task_id):
                 try:
                     verbose_task_start_callback(task)
                 except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)